Cycles Terms page (#30)
replit/replit-docs-2: docs/cycles/terms.md

## No refunds

Cycles are not refundable.

## Transfers

Cycles may not be transferred, bartered, sublicensed, or sold between users, except through authorized channels that Replit may provide in the future, to which further conditions may apply. Any violation of this condition may result in the forfeiture of your Cycles and/or the suspension or termination of your Replit account.

## No cash value

Cycles do not have any value in real currency, cannot be substituted for real currency, do not earn interest, and cannot be redeemed or refunded for real currency or anything else of value.

## Expiration and fees

Cycles do not expire for as long as you maintain your Replit account, and we do not charge no maintenance fees or dormancy fees. If your Replit account is terminated, you forfeit your Cycles balance, and you will have no recourse against Replit. Replit may also discontinue Cycles at any time, and no refunds will be issued.

## Customer service

> You may contact Replit with customer service inquiries about Cycles by email at [contact@replit.com](mailto:contact@replit.com).

Reviewer: ```suggestion
You may contact Replit with customer service inquiries about Cycles through [replit.com/support](https://replit.com/support).
```

we should use replit.com/support instead of email
